gbpservice/nfp/test/unit/configurator/agent/lb_test_agent.py

```diff
@@ -4,537 +4,197 @@
-                ],
-                "status_description": None,
-                "id": "6350c0fd-07f8-46ff-b797-62acd23760de",
-                "vip_id": "7a755739-1bbb-4211-9130-b6c82d9169a5",
-                "name": "lb-pool",
-                    "admin_state_up": True,
-                    "subnet_id": "b31cdafe-bdf3-4c19-b768-34d623d77d6c",
-                    "tenant_id": "f6b09b7a590642d8ac6de73df0ab0686",
-                    "health_monitors_status": [],
-                    "provider": "haproxy"}
-        return pool
-
-    def _fake_member_obj(self):
-        member = {
-            "admin_state_up": True,
-            "status": "ACTIVE",
-            "status_description": None,
-            "weight": 1,
-            "address": "42.0.0.11",
-            "tenant_id": "f6b09b7a590642d8ac6de73df0ab0686",
-            "protocol_port": 80,
-            "id": "4910851f-4af7-4592-ad04-08b508c6fa21",
-            "pool_id": "6350c0fd-07f8-46ff-b797-62acd23760de"}
-        return member
-
-    def _fake_hm_obj(self):
-        hm = {
-            "admin_state_up": True,
-            "tenant_id": "f6b09b7a590642d8ac6de73df0ab0686",
-            "delay": 10,
-            "max_retries": 3,
-            "timeout": 10,
-            "pools": [],
-            "type": "PING",
-                    "id": "c30d8a88-c719-4b93-aa64-c58efb397d86"
-        }
-        return hm
-
-    def _fake_kwargs(self):
-        kwargs = {'service_type': 'loadbalancer',
-                  'vm_mgmt_ip': '172.24.4.5',
-                  'mgmt_ip': '172.24.4.5',
-                  'source_cidrs': ['1.2.3.4/24'],
-                  'destination_cidr': ['1.2.3.4/24'],
-                  'gateway_ip': '1.2.3.4',
-                  'provider_interface_position': '1',
-                  'request_info': 'some_id',
-                  'rule_info': {
-                      'active_provider_mac': '00:0a:95:9d:68:16',
-                      'provider_mac': '00:0a:95:9d:68:16',
-                      'active_stitching_mac': '00:0a:95:9d:68:25',
-                      'stitching_mac': '00:0a:95:9d:68:25',
-                      'active_fip': '172.24.4.5',
-                      'fip': '172.24.4.5',
-                      'service_id': '1df1cd7a-d82e-4bbd-8b26-a1f106075a6b',
-                      'tenant_id': '6bb921bb81254b3e90e3d8c71a6d72dc'},
-                  'context': {'notification_data': 'hello'}
-                  }
-        return kwargs
-
-
-class ControllerTestCase(unittest.TestCase):
+class LBaasRpcSenderTest(unittest.TestCase):
+
+    @mock.patch(__name__ + '.FakeObjects.conf')
+    @mock.patch(__name__ + '.FakeObjects.sc')
+    def _get_configurator_rpc_manager_object(self, sc, conf):
+        cm = configurator.ConfiguratorModule(sc)
+        dmx = demuxer.ConfiguratorDemuxer()
+        rpc_mgr = configurator.ConfiguratorRpcManager(sc, cm, conf, dmx)
+        return sc, conf, rpc_mgr
+
+    def test_notifications(self):
```

Comment:
Don't we need to add test cases for update_status() update_pool_stats(), get_logical_device() ?